odata-csdl-xml/odata-csdl-xml-v4.02-csd01.md

```diff
@@ -0,0 +1,431 @@
+
+![OASIS Logo](https://docs.oasis-open.org/templates/OASISLogo-v3.0.png)
+
+-------
+
+# OData Common Schema Definition Language (CSDL) XML Representation Version 4.02
+
+## Committee Specification Draft 01
+
+## 14 July 2023
+
+&nbsp;
+
+#### This stage:
+https://docs.oasis-open.org/odata/odata-csdl-xml/v4.02/csd01/odata-csdl-xml-v4.02-csd01.md (Authoritative) \
```

Comment:
How about using placeholders for stage/previousstage?

Reply:
Will do, see file `odata-json-format/0 frontmatter.md`.

The starter documents need to be (manually?) split up into our source file structure.

Reply:
Done